src/config/dynamic-theme-fixes.config

```diff
@@ -2145,7 +2145,7 @@ button.section-directions-details-action-button
 div.section-loading-spinner
 a.gb_b > div
 a.gb_xc
-.gm-style img[role="presentation"]:not([src*="v=840"])
+.gm-style img[role="presentation"]:not([src*="v="])
```

Comment:
Wha- How did I didn't notice this in my `git diff`.
Is it like preloaded already? But the `display` declaration just changes?

Reply:
Not sure how it works... I saw in my dev tools in firefox change flashes in source when I changed to satellite and compared what I see fixes. I had v=871 in my maps but I think it can change again to something else. Still as it is still used in satellite view (not sure what it is - version or sth) then it should work until google change satellite map links to totally different 😉)

Reply:
What- I'm switching to Firefox dev tools lmao. Nothing was flashing in Chrome's one.

Reply:
Maybe because of this:
![obraz](https://user-images.githubusercontent.com/56877029/89226663-9d7da500-d5dc-11ea-8d1a-dc882f1c2277.png)

Reply:
Developer edition, it should be somewhere among the other browsers in my list 😂. Just need to get opera installed.